src/rgw/rgw_user.h

```diff
@@ -126,6 +126,7 @@ extern int rgw_remove_swift_name_index(RGWRados *store, string& swift_name);
 
 extern void rgw_perm_to_str(uint32_t mask, char *buf, int len);
 extern uint32_t rgw_str_to_perm(const char *str);
+extern uint32_t rgw_str_to_user_type(const char *str);
```

Comment:
IIRC we have an `enum` for user type, so we could use it instead of `uint32_t`.

The comment applies also to other places in the patch taking/returning user type.